package.json

```diff
@@ -31,11 +31,16 @@
       "email": "morenoh149@gmail.com",
       "url": "http://harrymoreno.com/"
     },
-    { "name": "Nick Pomfret" },
-    { "name": "rt2zz <zack@root-two.com>" }
+    {
+      "name": "Nick Pomfret"
+    },
+    {
+      "name": "rt2zz <zack@root-two.com>"
+    }
   ],
   "license": "MIT",
   "dependencies": {
+    "react-native": "^0.59.5",
```

Comment:
consider moving this to peerDependencies. Unless needed to make example app work, in which case it can go in devDependencies